src/Core/Framework/Script/Api/ScriptApiRoute.php

```diff
@@ -16,12 +16,15 @@
 use Symfony\Component\Routing\Annotation\Route;
 
 /**
- * @internal
```

Comment:
all route classes should be internal IMHO, we keep backwards compatibility on the HTTP-API level and not necessarily on the php class level